src/cmd/list.go

```diff
@@ -82,7 +82,7 @@ func init() {
 func list(cmd *cobra.Command, args []string) error {
 	if utils.IsInsideContainer() {
 		if !utils.IsInsideToolboxContainer() {
-			return errors.New("this is not a toolbox container")
+			return createErrorNonToolboxContainer()
```

Comment:
If the code in `preRun()` works as intended, then these duplicated checks in each of the commands might not be necessary.

Reply:
Check supressed.